### ISSUE TITLE
[5.5 The Density Matrix and Mixed States]: Some thoughts

### ISSUE THREAD

**Issue by dhruvbhq** [qiskit-community/qiskit-textbook#1122]
**Describe the problem**
I think some things can be added in this section for better clarity for first time readers.

**Describe the expected solution**
In [5.5.2 Mixed States](https://qiskit.org/textbook/ch-quantum-hardware/density-matrix.html#2.-Mixed-States-), The content states:
"In general, a mixed state consisting of an ensemble of n pure states can be expressed in the form of a list of outcome elements:" ... "where each item has a corresponding probability of occurrence given by".

Here, I think it would be good to _emphasize_ that these probabilities of occurrence are classical probabilities involved in a statistical ensemble, and do not refer to the post-measurement probabilities for quantum states. (i.e. that these probabilities are related to classical uncertainty in the state of the quantum system).

Further, I think we can mention that $n$ need not necessarily be the same as the dimension of the underlying Hilbert state.

Maybe @diemilio you would like to share your thoughts on this? :)

**Comment by diemilio** [comment]
Hi @dhruvbhq. I actually think this is a better place to keep track of issues in PR #514 because that has been closed and it will be difficult for others to find.

So, here is the list of issues/recommendations so far:

1. Tensor product notation in the chapter does not follow qiskit's qubit ordering.

2. The Reduced Density Matrix section may be rewritten to have a better flow and be less overwhelming. It seems that the section starts too "technical", which can be hard to follow for some, but then the example makes things clear. So the idea is to maybe try to start with the example, and then proceed with the formal definitions.

3. In 5.5.1 Pure States, the content states: "We understand that if we were to perform a measurement of this state, the outcome will be probabilisitc". It should be: "We understand that if we were to perform a measurement of this state, the outcome will be probabilistic", i.e. there's a typo in "probabilistic".

4. In 5.5.2 Mixed States, The content states:
"In general, a mixed state consisting of an ensemble of n pure states can be expressed in the form of a list of outcome elements:" ... "where each item has a corresponding probability of occurrence given by". Emphasize that these probabilities of occurrence are classical probabilities and that n need not necessarily be the same as the dimension of the underlying Hilbert space.

5. There's a type right above section 4.1: "In this specific example,  ρA and ρb are equal, but this is not always the case". The subscript b should be capitalized --> ρB.

**Comment by diemilio** [comment]
one more for the list:

6. Under Trace and Positivity Conditions: 
_2. The matrix must be positive-definite:_ should instead be _2. The matrix must be positive semi-definite:_

**Comment by shil-m** [comment]
Hi @diemilio @dhruvbhq  , I think I can help resolve some of the issues pointed out. Can I work on this?

**Comment by diemilio** [comment]
Hi @shil-m! 

Yeah, I will be OK if you take a shot at it. I was planning to do most myself, but I am having a hard time finding time to do it.

Just let me know once you complete them so I can start crossing them out on this list.

Thanks.

**Comment by shil-m** [comment]
@diemilio  In point 3, I am not sure what change are you expecting. I can't see a typo in "probabilistic".

**Comment by dhruvbhq** [comment]
I can see it is incorrectly written as "probabilisitc" (notice the last 4 characters 🙂 ).

**Comment by shil-m** [comment]
Thanks @dhruvbhq . It tricked my eyes somehow :)

**Comment by shil-m** [comment]
3, 4, 5, and 6 resolved in #1166

**Comment by diemilio** [comment]
Thanks @shil-m. For number 4, I am a bit concerned that "Hilbert space" is not introduced before in the qiskit textbook.

Here, $p_j$ represents classical probability and n can be anything not necessarily equal to the dimension of the underlying Hilbert Space.

How about:

Here, $p_j$ corresponds to the *classical* probability of the system being in state $|\psi_j \rangle$, and the total number of possible states $n$ does not need to be equal to the dimension of the underlying [Hilbert space](https://qiskit.org/textbook/ch-appendix/linear_algebra.html#Hilbert-Spaces,-Orthonormality,-and-the-Inner-Product).

**Comment by shil-m** [comment]
@diemilio This looks better indeed. I will change this.

**Comment by shil-m** [comment]
I forgot to mention, pls take a look #1166

**Comment by frankharkins** [comment]
Thanks for your contribution. I'm closing this issue as this repository is being archived. You can find the latest version of the Qiskit Textbook online at <https://qiskit.org/learn/>, and the new content repository at [Qiskit/textbook](https://github.com/Qiskit/textbook). Please feel free to re-open this issue there if it's still relevant.
